app/views/teaching_events/show/_provider-info.html.erb

```diff
@@ -0,0 +1,24 @@
+<% if [
```

Comment:
We need to be careful here not to show provider details for train to teach/question time events, even if they are present (which they are now so that we can display them in the structured data). See #2034 - would it make sense to have this logic in the presenter?